README.md

```diff
@@ -64,54 +82,60 @@ Potato aims to improve the replicability of data annotation and reduce the cost
 
 ### Dialogue analysis (span + categorization)
 
-    [launch] python3 potato/flask_server.py example-projects/dialogue_analysis/configs/dialogue-analysis.yaml -p 8000
+    [fetch] potato get dialogue_analysis
+    [launch] potato start dialogue_analysis -p 8000
     [Annotate] http://localhost:8000
 
 ![plot](./images/dialogue_analysis.gif)
 
 ### Sentiment analysis (categorization)
-
-    [launch] python3 potato/flask_server.py example-projects/sentiment_analysis/configs/sentiment-analysis.yaml -p 8000
+   
+    [fetch] potato get sentiment_analysis
+    [launch] potato start sentiment_analysis -p 8000
     [Annotate] http://localhost:8000
     
 ![plot](./images/sentiment_analysis.png)
     
 ### Summarization evaluation (likert + categorization)
 
-    [launch] python3 potato/flask_server.py example-projects/summarization_evaluation/configs/summ-eval.yaml -p 8000
+    [fetch] potato get summarization_evaluation
+    [launch] potato start summarization_evaluation -p 8000
     [Annotate] http://localhost:8000/?PROLIFIC_PID=user
     
 ![plot](./images/summ_eval.png)
 
 ### Match findings in papers and news (likert + prescreening questions + multi-task)
```

Comment:
I think we want to keep the main Readme.md a bit shorter. Could we move most of these examples to a separate page? (Maybe just keep the quickstart?). It would also help to see which of these are "baked" versus just examples people can start from.

Reply:
all these examples are available in example-projects.md, I can remove most of them here.

Reply:
fixed with [b083edf](https://github.com/davidjurgens/potato/pull/32/commits/b083edf373e81f08f8d7849add9c59f975988e1a)